packages/lib/src/vis/hooks.ts

```diff
@@ -75,7 +74,7 @@ export function useCameraState<T>(
 }
 
 export function useCustomColors(
```

Comment:
The hook could be renamed to something else: `useColorCssVariable` ?

Reply:
`useCssColors`?